README.md

```diff
@@ -15,93 +16,110 @@ Verified Telemetry (VT) is a state-of-the-art solution to determine the health o
 
 # Resource Requirements
+
 We expect Verified Telemetery code to run on microcontrollers, which have very limited amounts of flash and RAM. Below is our resource requirements to add verified telemetry code:
 
-| `Flash`       |  46 Kb  |																				  
+| `Flash`       |  46 Kb  |
 |---------------|---------|
-| `RAM`		    | 14 Kb   |
+| `RAM`      | 14 Kb   |
 
 # Plug and Play Model
+
 Verified Telemetry Library provides capabilities for interaction using a Plug and Play Model.
 Details about this model can be found [here](./PnPModel).
 
 # Usage Guide
 
 ## Implementing Platform level functions
+
 * For Verified Telemetry to work, it requires some platform specific functions to be defined at the application layer and then passed down to the library when VT is initialized.
 * Function type definitions can be found in [platform.h](./inc/platform/vt_platform.h)
 * Here is a summarised list of the functions that need to be defined by the application developer
-	| Platform Function 	| Requirements to be implemented by Application Developer                                	| Required? 	|
-	|-------------------	|----------------------------------------------------------------------------------------	|:---------:	|
-	| adc_init          	| Should initialize ADC when the controller and channel are passed as arguments          	|    YES    	|
-	| adc_read          	| Should return ADC data when the controller and channel are passed as arguments         	|    YES    	|
-	| gpio_on           	| Should set a GPIO pin HIGH when the GPIO port and pin are passed as arguments          	|    YES    	|
-	| gpio_off          	| Should set a GPIO pin LOW when the GPIO port and pin are passed as arguments           	|    YES    	|
-	| tick_init         	| Should initialize a tick counter with a time resolution that is passed as an argument  	|    YES    	|
-	| tick_deinit       	| Should de-initialize a running tick counter                                            	|    YES    	|
-	| tick              	| Should return the present tick value                                                   	|    YES    	|
-	| interrupt_enable  	| Should enable global interrupts on the MCU                                             	|  OPTIONAL 	|
-	| interrupt_disable 	| Should disable global interrupts on the MCU                                            	|  OPTIONAL 	|
+
+ | Platform Function  | Requirements to be implemented by Application Developer                                 | Required?  |
+ |------------------- |---------------------------------------------------------------------------------------- |:---------: |
+ | adc_single_init           | Should initialize ADC when the controller and channel are passed as arguments           |    YES     |
+ | adc_single_read           | Should return ADC data when the controller and channel are passed as arguments          |    YES     |
```

Comment:
>YES [](http://example.com/codeflow?start=124&length=3)

is this required even if they are just using current sense?

Reply:
modified